UI/RemoteISOScreen.cpp

```diff
@@ -198,51 +198,52 @@ static bool FindServer(std::string &resultHost, int &resultPort) {
 		return true;
 	}
 
-	// Start by requesting a list of recent local ips for this network.
-	if (http.Resolve(REPORT_HOSTNAME, REPORT_PORT)) {
-		if (http.Connect()) {
-			code = http.GET("/match/list", &result);
-			http.Disconnect();
+	if(!g_Config.bRemoteISOManual) {
```

Comment:
Change this to

    if (g_Config.bRemoteISOManual)
          return;

to avoid reindenting the rest of the function.

Also please add a comment explaining why.